api/src/main/java/org/openmrs/module/bedmanagement/service/impl/BedManagementServiceImpl.java

```diff
@@ -50,7 +50,6 @@ public void setDao(BedManagementDao dao) {
 		this.bedManagementDao = dao;
 	}
 	
-	@Autowired
 	public void setLocationService(LocationService locationService) {
```

Comment:
@SwathiVarkala is this setter actually still needed anyway?

Reply:
@mks-d Yes. 
It is required for setter based dependency injection.